components/system/components/Slider.js

```diff
@@ -0,0 +1,242 @@
+import * as React from "react";
+import * as Constants from "~/common/constants";
+
+import { css } from "@emotion/react";
+import { DescriptionGroup } from "~/components/system/components/fragments/DescriptionGroup";
+import { Input } from "~/components/system/components/Input";
+
+import Draggable from "react-draggable";
+
+const STYLES_BAR_CONTAINER = css`
+  height: 48px;
+  display: grid;
+  grid-template-columns: 1fr auto;
+  align-items: center;
+`;
+
+const STYLES_SLIDER_BAR = css`
+  width: calc(100% - 24px);
+  height: 4px;
+  border-radius: 2px;
+  background-color: ${Constants.system.gray};
+  position: absolute;
+  top: 10px;
+  left: 12px;
+  box-sizing: border-box;
+`;
+
+const STYLES_ACTIVE_SLIDER_BAR = css`
+  height: 4px;
+  border-radius: 2px 0px 0px 2px;
+  background-color: ${Constants.system.brand};
+  position: absolute;
```

Comment:
Does this need to be three absolute elements? Or just two?

Reply:
Imagine this can be done with one relative element, and two absolute elements. Bonus points if you do it with only one absolute element.